homeassistant/components/waterkotte_heatpump/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "waterkotte_heatpump",
+  "name": "Waterkotte Heatpump",
+  "codeowners": ["@chboland"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/waterkotte_heatpump",
+  "homekit": {},
+  "iot_class": "local_polling",
+  "requirements": ["pywaterkotte==0.1.2"],
+  "ssdp": [],
+  "zeroconf": []
```

Comment:
You can delete the unused ones

Reply:
redolved with [eab7ef6](https://github.com/home-assistant/core/pull/88140/commits/eab7ef65b5f3a23e0837922b0fbdb496ad21f652)